addon/package.json

```diff
@@ -36,6 +36,13 @@
   },
   "preferences": [
     {
+      "name": "WHITELIST_URLS",
```

Comment:
Not sure if this is what you meant @lmorchard. Seems odd to expose this to the user.